docs/reference/primitives/app/service.md

```diff
@@ -164,6 +167,29 @@ services:
 | --------- | ------ | ------- | ------------------------------------------------------------------------------------------ |
 | **cpu**     | number |         | The percentage of CPU utilization to target for [Processes](/reference/primitives/app/process) of this Service    |
 | **memory**  | number |         | The percentage of memory utilization to target for [Processes](/reference/primitives/app/process) of this Service |
+| **external**  | array of object |         | The array of the external metrics based on which it will scale the Service |
```

Comment:
```suggestion
| **external**  | map |         | The array of the external metrics based on which it will scale the Service |
```

We have other sections of the documentation using `map` for this kind of type.